### PR TITLE
Added hotkey to rename the QtConsole window.

### LINKED ISSUE #100: Add API to change QtConsole title
I often have multiple QtConsole windows open, and it gets very hard to tell them apart.
It would be great if there was an API to allow changing the window title.

### PR DESCRIPTION
**Pull request by tmr232**
The <kbd>Alt+R</kbd> hotkey now renames the main QtConsole window.
This closes #100 .

### CODE REVIEW

File: qtconsole/mainwindow.py [hunk 1]
```diff
@@ -98,7 +98,17 @@ def create_tab_with_new_frontend(self):
...
         widget = self.new_frontend_factory()
         self.add_tab_with_frontend(widget)
-    
+
+    def set_window_title(self):
+        """Set the title of the console window"""
+        old_title = self.windowTitle()
+        title, ok = QtGui.QInputDialog.getText(self,
+                                               "Rename Window",
+                                               "New Title:".format(old_title),
```
Comment (wmvanvliet) [inline qtconsole/mainwindow.py]: unnecessary code. New Title -> New title.

File: qtconsole/mainwindow.py [hunk 2]
```diff
@@ -598,7 +608,14 @@ def init_window_menu(self):
             statusTip="Select next tab",
             triggered=self.next_tab)
         self.add_menu_action(self.window_menu, self.next_tab_act)
-    
+
+        self.rename_window_act = QtGui.QAction("Rename &Window",
+                                                    self,
+                                                    shortcut="Alt+R",
+                                                    statusTip="Rename window",
+                                                    triggered=self.set_window_title)
```
Comment (ccordoba12) [inline qtconsole/mainwindow.py]: Please align these lines to the left parenthesis of line `612` to follow pep8

**Comment by ccordoba12** [comment]
+1 from me for this one :-)

**Comment by minrk** [comment]
Looks like this conflicts with #148, but ok to merge once conflicts are resolved.

**Pull request merged**